src/components/genetic-variants/genetic-variants.tsx

```diff
@@ -454,17 +496,25 @@ export class GeneticVariants implements ComponentInterface {
                 {this.visibleComponents().map(component =>
                   <td key={component.system + '/' + component.code}>
                     {this.getComponentValues(resource, component.expression).map(value =>
-                      <span key={value}>
+                      <span key={value}>                          
                         {(component.link && component.link.url && component.link.attribute ) ? 
                           (<span>
                           <a href={this.createLink(component, value)} target="_blank">{ value }</a> 
                           &#32;
                           </span>)
                         : <span>{ value }&#32;</span>}                    
                       </span>
-                    )}                  
+                    )}
                   </td>
                 )}
+                {this.showVariantBrowser ? 
+                  <td>
+                    <span>
+                      <a href={this.getVariantBrowserURL(resource)} target="_blank"> {this.localeString.openVariantBrowser} </a> 
```

Comment:
Replace "Variant Browser öffnen" with VB in all languages. Add a tooltip to the text (can be title attribute) with the long text ("Variant Browser öffnen")